### ISSUE TITLE
Error: map::at 

### ISSUE THREAD

**Issue by raagagrawal** [uclahs-cds/public-R-SeqKat#3]
Running SeqKat with the following command results in the error: 

Error: map::at

Command:
seqkat(
    sigcutoff = 5, 
    mutdistance = 3.2, 
    segnum = 4, 
    ref.dir = '/hot/user/ragrawal/reference/GRCh38-BI-20160721/', 
    bed.file = example.bed.file, 
    output.dir = "/hot/user/ragrawal/Liss-Agrawal-PRAD-COLB/kataegis/seqkat_output", 
    chromosome = "4", 
    chromosome.length.file = '/hot/user/ragrawal/Liss-Agrawal-PRAD-COLB/kataegis/seqkat_input/chromosome_lengths.txt', 
    trinucleotide.count.file = '/hot/user/ragrawal/Liss-Agrawal-PRAD-COLB/kataegis/seqkat_input/trinucleotide_counts.txt')

Bed file is from the package itself: 
example.bed.file <- paste0(
	path.package("SeqKat"),
	"/extdata/test/PD4120a-chr4-1-2000000_test_snvs.bed"
	);

Chromosome length file looks like this:
num	length
1	248956422
2	242193529
3	198295559
4	190214555
5	181538259
6	170805979
7	159345973
8	145138636
9	138394717
10	133797422
11	135086622
12	133275309
13	114364328
14	107043718
15	101991189
16	90338345
17	83257441
18	80373285
19	58617616
20	64444167
21	46709983
22	50818468
23	156040895
24	57227415
sum.f	3031042417
sum.m	3088269832

Trinucleotide Count file looks like this:
trinucleotide count
AAA 111989738
AAC	43201194
AAG	58189769
AAT	72371689
ACA	58990692
ACC	33696401
ACG	7493615
ACT	47217592
AGA	65098960
AGC	40871572
AGG	51544445
AGT	47153556
ATA	60052604
ATC	38913595
ATG	53392579
ATT	72972218
CAA	55028101
CAC	43811665
CAG	59422990
CAT	53647632
CCA	53187843
CCC	37938793
CCG	7997076
CCT	51686199
CGA	6476677
CGC	6973826
CGG	8163775
CGT	7538164
CTA	37526148
CTC	49233040
CTG	58845855
CTT	58949909
GAA	58550174
GAC	27584997
GAG	49296690
GAT	39318178
GCA	42251785
GCC	34400705
GCG	7021186
GCT	40553684
GGA	45711130
GGC	34385793
GGG	38041000
GGT	33680960
GTA	33111127
GTC	27376011
GTG	44271034
GTT	42931358
TAA	60184362
TAC	32800448
TAG	37758992
TAT	59993479
TCA	57480061
TCC	44773999
TCG	6640570
TCT	65097477
TGA	57463257
TGC	41996189
TGG	54069670
TGT	59316856
TTA	60047401
TTC	58469576
TTG	56336500
TTT	113221598

**Comment by tyamaguchi-ucla** [comment]
@raagagrawal I think the example was probably build with GRCh37 while GRCh38 was used as reference.

**Comment by raagagrawal** [comment]
@tyamaguchi-ucla Yep! Could be the explanation for the example.bed. This also fails w/ the same error code with my sample .bed file. Path: /hot/user/ragrawal/Liss-Agrawal-PRAD-COLB/kataegis/seqkat_input/BCLB_snvs.bed.
